tests/format/js/pipeline-operator/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -240,14 +240,14 @@ async function * f () {
 a |> (await %) |> % * 3;
 
 foo
-  |> (await %)
-  |> % || throw new Error(\`foo \${bar1}\`)
-  |> bar2(%, ", ")
-  |> bar3(%)
-  |> % + "!"
-  |> new Bar.Foo(%)
-  |> (await bar.bar(%))
-  |> console.log(%);
+|> (await %)
+|> % || throw new Error(\`foo \${bar1}\`)
+|> bar2(%, ", ")
+|> bar3(%)
+|> % + "!"
+|> new Bar.Foo(%)
+|> (await bar.bar(%))
+|> console.log(%);
```

Comment:
I'm not sure the best way, but following [official explainer](https://github.com/tc39/proposal-pipeline-operator#real-world-examples), we don't add indent on here.

Reply:
It's clear for me to indent.

Reply:
I prefer no indentation. Also other languages code formatter outputs following:

## Elixir (by `mix format`)

**Input:**

```ex
100 |> foo(1) |> bar(2) |> baz(3) |> bar(2) |> baz(3) |> bar(2) |> baz(3)
```

**output:**

```ex
100
|> foo(1)
|> bar(2)
|> baz(3)
|> bar(2)
|> baz(3)
|> bar(2)
|> baz(3)
```

## F# (by [Fantomas](https://github.com/fsprojects/fantomas))

**input:**

```fs
100 |> function1 |> function2 |> function1 |> function2 |> function1 |> function2 |> function1 |> function2 
```

**output:**

```fs
100
|> function1
|> function2
|> function1
|> function2
|> function1
|> function2
|> function1
|> function2
```

Reply:
This can't convince me. I'd like to hear opinions from other maintainers.

Reply:
Okay, I have no strong opinion for this. So for now, let's use indentation version, same as current Prettier.